data-migrator/core/src/main/java/io/camunda/migration/data/impl/interceptor/history/entity/DecisionRequirementsDefinitionTransformer.java

```diff
@@ -46,15 +62,100 @@ public void execute(EntityConversionContext<?, ?> context) {
 
     String deploymentId = c7DecisionRequirements.getDeploymentId();
     String resourceName = c7DecisionRequirements.getResourceName();
-    String dmnXml = c7Client.getResourceAsString(deploymentId, resourceName);
 
+    String dmnXml = prefixDecisionIdsInDmn(Dmn.readModelFromStream(c7Client.getResourceAsStream(deploymentId, resourceName)));
```

Comment:
If we don't add prefixes to the DMN XML, Operate cannot link executed decision instances badges.